.github/workflows/unit_test_200gb_CI.yml

```diff
@@ -137,6 +137,8 @@ jobs:
           export VESPA_CONFIG_URL=http://localhost:19071
           export VESPA_DOCUMENT_URL=http://localhost:8080
           export VESPA_QUERY_URL=http://localhost:8080
+          export MARQO_MAX_CPU_MODEL_MEMORY=15
```

Comment:
Let's not change this one because we want the CPU unit test to be light weight. Can you move your full model test to large model test?